Dockerfile

```diff
@@ -8,4 +8,4 @@ COPY . .
 
 EXPOSE 8080
 
-CMD bash -c "flask db upgrade && gunicorn -w 1 -b 0.0.0.0:8080 app:create_app()"
+CMD ["gunicorn", "-w", "1", "-b", "0.0.0.0:8080", "app:create_app()"]
```

Comment:
This shouldn't be needed, or changed?

We don't want to mess with the Dockerfile, as that's used by both processes - it should not needed to be changed at all to allow it to run for paketo

Reply:
Removed